From fb043a27324ba81502b8986a31222f38aa414bbf Mon Sep 17 00:00:00 2001
From: Miaoqing Pan <miaoqing.pan@oss.qualcomm.com>
Date: Thu, 18 Dec 2025 09:46:03 +0800
Subject: [PATCH 1/3] defconfig: Update Opportunistic Wireless Encryption (OWE)
 state

OWE enhances privacy in public and enterprise environments where open
networks are prevalent. Enabling OWE aligns with modern security best
practices and supports the testing and development of OWE-capable
devices.

OWE is now standardized in IEEE Std 802.11-2024 while it was originally
specified in IETF RFC 8110 (updated by RFC 9672). It is not experimental
anymore, i.e., there has been significant interoperability testing and
there are deployed cases.

Signed-off-by: Miaoqing Pan <miaoqing.pan@oss.qualcomm.com>
Upstream-Status: Backport [https://git.w1.fi/cgit/hostap/commit/?id=39db92dcf301793ce45a8ebf85c425f67c670058]
---
 wpa_supplicant/defconfig | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/wpa_supplicant/defconfig b/wpa_supplicant/defconfig
index 52befd8..044604a 100644
--- a/wpa_supplicant/defconfig
+++ b/wpa_supplicant/defconfig
@@ -638,8 +638,7 @@ CONFIG_BGSCAN_SIMPLE=y
 #CONFIG_BGSCAN_LEARN=y
 
 # Opportunistic Wireless Encryption (OWE)
-# Experimental implementation of draft-harkins-owe-07.txt
-#CONFIG_OWE=y
+CONFIG_OWE=y
 
 # Device Provisioning Protocol (DPP) (also known as Wi-Fi Easy Connect)
 CONFIG_DPP=y
-- 
2.34.1

